index.html

```diff
@@ -99,17 +99,34 @@ <h3>Extensions to the <code>Window</code> interface</h3>
           <dl>
             <dt><dfn data-dfn-for="Window">visualViewport</dfn></dt>
             <dd>
-              If the <code>window</code> has a <dfn data-cite="!HTML#browsing-context">Browsing Context</dfn> set,
-              return the <a>VisualViewport</a> object associated with it. Otherwise, return null.
+              If the <a>window</a> is currently in a <dfn data-cite="!HTML#windowproxy">WindowProxy</dfn>
+              object's <dfn data-cite="!HTML#concept-windowproxy-window">[[\Window]]</dfn> internal slot, return the
+              <a>VisualViewport</a> object associated with it. Otherwise, return null.
+              <p class="note">
+                Intuitively, the VisualViewport object is only returned and useful for a <a>window</a> whose
+                <dfn data-cite="!HTML#document">Document</dfn> is currently being presented.
+                If a reference is retained to a <a>VisualViewport</a> that's not in a currently presented
+                <a>Document</a>, the values in that <a>VisualViewport</a> must not reveal any
+                information about the <dfn data-cite="!HTML#browsing-context">Browsing Context</dfn>.
+
+              <p>
             </dd>
           </dl>
         </section>
         <section data-dfn-for="VisualViewport">
           <h3>The <dfn><code>VisualViewport</code></dfn> interface</h3>
           <p>
-            A <a>VisualViewport</a> object represents the visual viewport for a given <code>browsing context</code>.
-            Each <code>window</code> on a page will have a unique <a>VisualViewport</a> object representing
-            the properties associated with the <code>window</code>'s <code>browsing context</code>.
+            A <a>VisualViewport</a> object represents the visual viewport for a given <a>window</a>.
+            Each <a>window</a> on a page will have a unique <a>VisualViewport</a> object. This object
+            represents the properties of the <a>Browsing Context</a> whose <a>WindowProxy</a> object's
+            <a>[[\Window]]</a> internal slot contains the <a>window</a>. If the <a>window</a>
+            does not currently occupy a <a>[[\Window]]</a> internal slot, it must return 0 or null for
```

Comment:
Is 0 the fallback value we want for scale, too (which is something you typically multiply or divide by)?

Reply:
Chrome currently does return 0 for scale. These values aren't meant to be used when the associated document isn't being presented -- and there really isn't any scale in that case -- so I consider 0 reasonable. I don't feel particularly strongly though if you think there's a compelling reason to change it.